plugins/browser-plugin-snowplow-ecommerce/src/api.ts

```diff
@@ -162,9 +162,19 @@ export function trackTransaction(
   transaction: Transaction & CommonEcommerceEventProperties,
   trackers: Array<string> = Object.keys(_trackers)
 ) {
+  let totalQuantity = 0;
   const { context = [], timestamp, products = [], ...transactionAttributes } = transaction;
-  products.forEach((product) => context.push({ schema: PRODUCT_SCHEMA, data: product }));
-  context.push({ schema: TRANSACTION_SCHEMA, data: { ...transactionAttributes } });
+  products.forEach((product) => {
+    /* If `total_quantity` is not provided, we calculate it from individual product quantities. */
+    if (product.quantity) {
+      totalQuantity += product.quantity;
+    }
+    context.push({ schema: PRODUCT_SCHEMA, data: product });
+  });
+  context.push({
+    schema: TRANSACTION_SCHEMA,
+    data: { total_quantity: totalQuantity || undefined, ...transactionAttributes },
```

Comment:
Great question! In this case, I would expect the validation to fail, since a transaction without any kind of quantity information should count as invalid. WDYT ?

Reply:
Makes sense! 👍